tests/lib/rules/no-shadow-route-definition.js

```diff
@@ -92,6 +92,11 @@ ruleTester.run('no-shadow-route-definition', rule, {
       this.route("post");
     });
     this.route("post");`,
+    // See issue: https://github.com/ember-cli/eslint-plugin-ember/issues/1037
```

Comment:
Instead of linking to the issue, can you use this comment to describe what is being tested in this case?

Reply:
Done.